Demos/LightboxDemo/LightboxDemo/ViewController.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+import Lightbox
+
+class ViewController: UIViewController {
+
+  lazy var lightboxView: LightboxView = { [unowned self] in
```

Comment:
@richardoti To avoid strong reference cycle between the closure and the instance. You could read about it here for example http://sketchytech.blogspot.no/2014/09/swift-rules-of-weak-and-unowned.html